code/modules/antagonists/roguetown/villain/zizocult.dm

```diff
@@ -186,7 +186,7 @@ GLOBAL_LIST_EMPTY(ritualslist)
 	set name = "Communicate"
 	set category = "ZIZO"
 
-	if(stat == DEAD)
+	if(src.stat == DEAD)
```

Comment:
same as above

Reply:
it is simply not working, game simply does not know whose dead stat it needs to check